src/applications/personalization/preferences/constants.js

```diff
@@ -1,9 +1,5 @@
+export { LOADING_STATES } from '../common/constants';
```

Comment:
Many files are importing the LOADING_STATES from this file so rather than update all those files I'm just doing an `export ... from` where they used to be defined.